block1_bugrace_intro/README.md

```diff
@@ -2,7 +2,7 @@
 
 Nu ska du få lära känna Scratch och samtidigt skapa ett roligt spel som är enkelt även om du aldrig har jobbat med Scratch tidigare. Målet är att få en skalbagge att springa runt en bana och försöka undvika att hamna utanför banan genom att styra med vänster- och högerpil på tangentbordet.
 Ett exempel på det färdiga spelet finns på: [https://scratch.mit.edu/projects/27697024/](https://scratch.mit.edu/projects/27697024/)
-<a href="www.dn.se" target="_blank">test</a>
+<br><a href="http://www.dn.se" target="_blank">test</a>
```

Comment:
Länkarna får ni ha kvar i html om de ska öppnas i nytt fönster, går inte att lösa i markdown.

Vi kan använda funktionaliteten för blockquote för att få in en ruta. Gör så här:

Skriv en ”>” framför stycket som ska ha en ruta runt sig. Om ni vill ha rubrik så skriv två mellanslag på raden efter rubriken och ett enterslag, då hamnar allt i samma ruta. så här alltså:


> **rubrik**  
Text som ska stå i rutan


Palle

> On 4 dec. 2015, at 09:43, tinavincent <notifications@github.com> wrote:
> 
> In block1_bugrace_intro/README.md:
> 
> > @@ -2,7 +2,7 @@
> >  
> >  Nu ska du få lära känna Scratch och samtidigt skapa ett roligt spel som är enkelt även om du aldrig har jobbat med Scratch tidigare. Målet är att få en skalbagge att springa runt en bana och försöka undvika att hamna utanför banan genom att styra med vänster- och högerpil på tangentbordet.
> >  Ett exempel på det färdiga spelet finns på: [https://scratch.mit.edu/projects/27697024/](https://scratch.mit.edu/projects/27697024/)
> > -<a href="www.dn.se" target="_blank">test</a>
> > +<br><a href="http://www.dn.se" target="_blank">test</a>
> 
> Jag kan ändra, men då återstår två saker: 1. Hur skriver man länkar som öppnas i ett nytt fönster mha markdown? Motsvarande <a target="_blank"> 2. Hur ska vi markera det som hittat nu ligger i en html-tabell enligt Lisas önskemål? /Tina
> …
> —
> Reply to this email directly or view it on GitHub.
>